### ISSUE TITLE
Error: command error: failed to load modules: module xxx/xxx not found in  either add it as a Hugo Module or store it in "xxx/themes".: module does not exist

### ISSUE THREAD

**Issue by wzdushu** [xianmin/hugo-theme-jane#400]
Hi:

After I upgraded the theme to the latest version V3, I encountered an error， as follows:

Error: command error: failed to load modules: module "github.com/hugomods/icons/vendors/lucide" not found in "/Users/wanzi/github/wzdushu/wnote/themes/github.com/hugomods/icons/vendors/lucide"; either add it as a Hugo Module or store it in "/Users/wanzi/github/wzdushu/wnote/themes".: module does not exist

Then I executed the following operations without any problem, but the same error occurred when starting hugo. Is it because of my path problem?：
```
cd /Users/wanzi/github/wzdushu/wnote/themes/jane
hugo mod get github.com/hugomods/icons/vendors/mdi
hugo mod get github.com/hugomods/icons/vendors/lucide
```

My hugo and golang version:
```
hugo v0.136.0+extended darwin/arm64 BuildDate=2024-10-15T13:17:24Z VendorInfo=brew
go1.23.2 darwin/arm64
```

My hugo project directory structure：

```
> pwd
/Users/wanzi/github/wzdushu/wnote
> tree -L 2                                                                                                                  
.
├── archetypes
│   └── default.md
├── config.toml
├── content
│   ├── about.md
│   ├── notes.md
│   └── post
├── data
├── layouts
├── public
├── static
└── themes
    └── jane

10 directories, 4 files
```

**Comment by razonyang** [comment]
You will need to initialize Hugo Modules (`go.mod`), and then Hugo will start downloading the required modules when building the site and starting the server.

```sh
$ cd /path/to/yoursite
$ hugo mod init github.com/yourname/yourrepo
$ hugo server
```

**Comment by xianmin** [comment]
Thank you very much for your feedback. This was indeed an oversight on my part, and I will update the README.md file as soon as possible.

**Comment by xianmin** [comment]
The current issue is a bit beyond my expectations.
I didn't consider that using Hugo modules would require installing the Go language environment, as I already had Go installed on my computer. 
If this is the case, I might need to think about it more carefully. 
It's possible that removing the Hugo modules might be a better option... I don't want installing Go to be a must choice.

Give me some time to consider how to better handle this issue.

**Comment by wzdushu** [comment]
I referred to your Github action configuration, I lowered my Hugo version, and now I can start the server and generate static pages normally.

It is indeed necessary to update the read document, which must rely on the Go module. In addition, you need to pay attention to the go version and hugo version.

**Comment by wzdushu** [comment]
> 您需要初始化 Hugo 模块（`go.mod`），然后 Hugo 将在构建站点和启动服务器时开始下载所需的模块。
> 
> ```shell
> $ cd /path/to/yoursite
> $ hugo mod init github.com/yourname/yourrepo
> $ hugo server
> ```

Yeah, I know I need to initialize the go module. I tried it but it didn’t work. Finally, I lowered the hugo version to 0.121.0 to keep it consistent with the author.

**Comment by xianmin** [comment]
fixed in 35c16043ca4169ba2bcc4dc30c9a4ed8802649cb . Now, we don't need hugo modules.
